docs/cookbook.rst

```diff
@@ -0,0 +1,138 @@
+The Nox Cookbook
+================
+
+The What?
+---------
+
+A lot of people and a lot of projects use ``nox`` for their python automation powers.
+
+Some of these sessions are the classic "run pytest and linting", some are more unique and more interesting!
+
+The nox cookbook is a collection of these such sessions.
+
+``nox`` is super easy to get started with, and super powerful right out of the box. But when things get complex or you want to chain together some more powerful tasks, often the only examples can be found hunting around GitHub for novel ``nox`` sessions.
```

Comment:
Nit: use "Nox" when referring to the project, only use ``nox`` when referring to the command line program. Also, Nox is capitalized (unlike tox).